backend/src/executors.py

```diff
@@ -72,23 +82,23 @@ def __init__(self, **kwargs):
 
             # # Log success on successful save
             print(
-                "[ProtBertExecutor.__init__] SUCCESS: Tokenizer saved"
+                "[ProtBertExecutor.initialize_executor] SUCCESS: Tokenizer saved "
                 "successfully to local"
             )
 
         # Assign model from cache
-        self.model = BertModel.from_pretrained("./models/prot_bert")
+        ProtBertExecutor.__model = BertModel.from_pretrained("./models/prot_bert")
 
         # Success logging
-        print("[ProtBertExecutor.__init__] SUCCESS: Loaded model from cache")
+        print("[ProtBertExecutor.initialize_executor] SUCCESS: Loaded model from cache")
 
         # Assign tokenizer from cache
-        self.tokenizer = BertTokenizer.from_pretrained(
+        ProtBertExecutor.__tokenizer = BertTokenizer.from_pretrained(
             "./tokenizers/prot_bert", do_lower_case=False
         )
 
         # Success logging
-        print("[ProtBertExecutor.__init__] SUCCESS: Loaded tokenizer from cache")
+        print("[ProtBertExecutor.initialize_executor] SUCCESS: Loaded tokenizer from cache")
```

Comment:
Do we need to do this in advance? Unless loading the model from cache is an expensive operation (i.e. it requires seconds/minutes) I would avoid sharing the extra variables `__model` and `__tokenizer` among different methods. We can load the model and the tokenizer in the constructor and use the standard `self.model` and `self.tokenizer`.

Reply:
Yes.

The only concern this addresses is the first time for `make docker` where no cache exists. I was afraid of breaking changes, and I think the double-underscore will not be accessible from an object instantiation.

The constructor throws a `timeout` error due to `Flow` not being able to load the `ProtBertExecutor` when the model is being fetched (the first time, with no cache). To prevent any and all possible timeout logs, because the users will have varying bandwidths, I went with the approach of keeping private variables (only in the class) just to make sure the `self.*` are properly instantiated and loaded without any `timeout` problems.